inc/Test/Net/SSLeay.pm

```diff
@@ -466,6 +607,46 @@ this Perl, or false if not.
 Returns the relative path to a given file in the test suite data directory
 (C<t/local/>). Dies if the file does not exist.
 
+=head2 dies_like
```

Comment:
The commit comment has an excellent description of what is implemented by `dies_like()` and the other new functions. Should the documentation for these new functions also briefly mention what they reimplement. That would be useful to inform the reader that they should not consider using Test::Exception and the other modules but look more closely what these functions do.